modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/Interpolator1D.java

```diff
@@ -20,30 +20,26 @@
  * A base class for interpolation in one dimension.
  */
 public abstract class Interpolator1D
-    implements Interpolator<Interpolator1DDataBundle, Double>, Serializable {
+    implements Serializable {
```

Comment:
Does an interpolator really need to implement `Serializable`? Particularly given it's not a bean.